charter.md

```diff
@@ -31,9 +31,8 @@ QUIC datagrams) and can be used over raw QUIC or WebTransport.
 The proposed solution will provide extensibility for supporting different media
 formats and shall specify a mandatory to implement media format to ensure
 interoperability. Support for multiple media types and media encodings shall be
-proposed. The solution will specify a simple method of authentication to access
-media, as well as a mechanism for carrying information enabling additional
-decryption of media payloads where required.
+proposed. The solution will specify a simple method for clients to authenticate
```

Comment:
I have seen mail thread on justifying why 'simple' is there. But still suggest to remove it. No solution starts with 'i-will-be-complex'. if there's a specific reason, it should be mentioned instead.